_posts/francois_gerthoffert/2019-03-03-ZenCrepes-GitHub-Agile-Analytics-made-easy.md

```diff
@@ -0,0 +1,301 @@
+
+As soon as you begin operating on more than one repository, there is a lack of tools to facilitate management at an organization level or even across organizations. GitHub Projects at an organization level are limited to 5 repositories per project, labels and milestones are unique per repositories... 
+
+ZenCrepes was created to address this limitation, by providing the missing cross-org & cross-repos view.
+
+# Key Features
+
+It focuses on three primary objectives:
+* __Report and search__: Quickly find relevant issues based on selected criteria (faceted search). For example, “List open defects assigned to John or Max in rock or paper repos” or “Display the team's overall velocity on paper and scissor repos”.
+* __Scrum operation__: Provide one scrum view to identify the amount of work left in a sprint, estimate completion based on past velocity, review repartition of open issues (by repo, by labels, by assignees).
+* __Consistency__: Ensure labels and milestones are consistent across multiple organizations and repositories, clean-up when necessary.
+
+
+# Report and search
+
+For those familiar with our portals, no need to search long to understand where the inspiration is coming from for the “Issues” view (no pie charts though!).
+
+<figure>
+    <img src="{{site.urlimg}}francois_gerthoffert/zencrepes/report-and-search.png" />
+    <figcaption>Summary view in the issues tab</figcaption>
+</figure>
+
+Using faceted search users can build queries to filter down on elements being analyzed and displayed, providing views into the operation of multiple teams. 
+
+<figure>
+    <img src="{{site.urlimg}}francois_gerthoffert/zencrepes/report-and-search-list-view.png" />
+    <figcaption>List view in the Issues tab</figcaption>
+</figure>
+
+The list view contains the list of all issues resulting from the query. Clicking on the issue opens it in GitHub for further investigations/modifications.
+
+Two more tabs (Velocity & Burndown) are also available to view those metrics over the entire duration of the project.
+
+
+## Find the odd status
+
+Do we have any issues, part of closed milestones, which are still open? This shouldn’t be possible, right? Let’s check it out!
+
+<figure>
+    <img src="{{site.urlimg}}francois_gerthoffert/zencrepes/report-and-search-odd.png" />
+    <figcaption>Something to investigate</figcaption>
+</figure>
+
+So out of the ~3500 issues, 5 are in an incorrect state and worth further investigations. Those were last updated almost a year ago, likely forgotten by their team.
+
+
+## Assembling a team
+
+
+__Note of caution__: _The elements detailed below only works for teams, used at working together with a similar process, and should only be considered as one of the possible metrics (or as a starting point), but shouldn't be considered as an accurate metric._
```

Comment:
Correct to "...used **for** working together..."

Reply:
really? I wanted to refer to the team composed of people who have been working together. Should I use "for" ?

Reply:
You can ignore this, I understand now!